crawler.sh

```diff
@@ -0,0 +1,11 @@
+export PYTHONPATH="$(pwd)"
```

Comment:
we should move this file to somewhere into a directory instead of at the root of the repo